libdevice/sanitizer/msan_rtl.cpp

```diff
@@ -219,6 +219,14 @@ inline void __msan_exit() {
     __devicelib_exit();
 }
 
+template <typename dataT>
```

Comment:
```suggestion
template <typename T>
```
nit: I think `T` would be enough.